src/main/java/ip/task/Event.java

```diff
@@ -0,0 +1,16 @@
+package ip.task;
+
+public class Event extends Task{
+
+    protected String at;
+
```

Comment:
Perhaps consider renaming this variable to date/deadline to be a bit more precise.

Reply:
I disagree here. `Deadline` is already the name of the object, and the deadline might be relative and not a date.